pkgs/development/tools/locate-dominating-file/default.nix

```
@@ -0,0 +1,48 @@
+{
+  bats,
+  bash,
+  lib,
+  stdenv,
+  getopt,
+  fetchFromGitHub,
+}: let
+  batsPkg = bats.withLibraries (p: [p.bats-support p.bats-assert]);
+  version = "v0.0.1";
+in
+  stdenv.mkDerivation {
```

```suggestion
stdenv.mkDerivation {
```

`in` keyword does not start an indentation.